CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+**Features**:
+- Add inbound filters functionality to dynamic sampling rules. ([#920](https://github.com/getsentry/relay/pull/920))
```

Comment:
Let's move this to `Internal` again. Dynamic sampling is not an exposed feature yet. This aligns with how we communicated the past dynamic sampling PRs if we compare with the last releases.

Reply:
ok